docs/sql/general/constraints.rst

```diff
@@ -53,6 +53,96 @@ constraint or a table constraint.
 For further details about the meaning of the options see
 :ref:`indices_and_fulltext`.
 
+.. _check_constraint:
+
+``CHECK``
+---------
+
+The CHECK constraint specifies that the value/s of certain column/s must satisfy
```

Comment:
As above

Reply:
thank you! both corrected to plural then.